pre_processing/dependencies.inc

```diff
@@ -66,8 +66,9 @@ $(OBJS)/read_avhrr.o: $(OBJS)/calender.o $(OBJS)/channel_structures.o \
         read_avhrr_time.F90 read_avhrr_time_lat_lon_angles.F90
 $(OBJS)/read_camel_emissivity.o: $(OBJS)/preproc_constants.o
 $(OBJS)/read_cimss_emissivity.o: $(OBJS)/preproc_constants.o
-$(OBJS)/read_himawari.o: $(OBJS)/channel_structures.o \
-        $(OBJS)/imager_structures.o $(OBJS)/preproc_constants.o
+$(OBJS)/read_himawari.o: $(OBJS)/calender.o $(OBJS)/channel_structures.o \
+        $(OBJS)/imager_structures.o $(OBJS)/preproc_constants.o \
+        $(OBJS)/solar_position.o
```

Comment:
Likewise this one, was something broken when we were missing solar_position?

Reply:
This was automatic, seeming because of https://github.com/ORAC-CC/orac/blob/c3cacad2266d9666a502bd29c5a67e9e84d2f75a/pre_processing/read_himawari.F90#L76 and https://github.com/ORAC-CC/orac/blob/c3cacad2266d9666a502bd29c5a67e9e84d2f75a/pre_processing/read_himawari.F90#L181